templates/js/custom-copilot-basic/README.md.tpl

```diff
@@ -0,0 +1,101 @@
+# Overview of the Basic AI Chatbot template
+
+This template showcases a bot app that responds to user questions like ChatGPT. This enables your users to talk with the AI bot in Teams.
+
+The app template is built using the Teams AI library, which provides the capabilities to build AI-based Teams applications.
+
+- [Overview of the Basic AI Chatbot template](#overview-of-the-basic-ai-chatbot-template)
+  - [Get started with the Basic AI Chatbot template](#get-started-with-the-basic-ai-chatbot-template)
+  - [What's included in the template](#whats-included-in-the-template)
+  - [Extend the Basic AI Chatbot template with more AI capabilities](#extend-the-basic-ai-chatbot-template-with-more-ai-capabilities)
+  - [Additional information and references](#additional-information-and-references)
+
+## Get started with the Basic AI Chatbot template
+
+> **Prerequisites**
+>
+> To run the Basic AI Chatbot template in your local dev machine, you will need:
+>
+> - [Node.js](https://nodejs.org/), supported versions: 16, 18
+{{^enableTestToolByDefault}}
+> - A [Microsoft 365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+{{/enableTestToolByDefault}}
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-toolkit-cli)
+{{#useOpenAI}}
+> - An account with [OpenAI](https://platform.openai.com/)
+{{/useOpenAI}}
+{{#useAzureOpenAI}}
+> - Prepare your own [Azure OpenAI](https://aka.ms/oai/access) resource.
+{{/useAzureOpenAI}}
+
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+{{#enableTestToolByDefault}}
+{{#useOpenAI}}
+1. In file *env/.env.testtool.user*, fill in your OpenAI key `SECRET_OPENAI_API_KEY=<your-key>`.
+{{/useOpenAI}}
+{{#useAzureOpenAI}}
+1. In file *env/.env.testtool.user*, fill in your Azure OpenAI key `SECRET_AZURE_OPENAI_ENDPOINT=<your-key>` and endpoint `SECRET_AZURE_OPENAI_ENDPOINT=<your-endpoint>`.
+1. In `src/app/app.js`, update `azureDefaultDeployment` to your own model deployment name.
+{{/useAzureOpenAI}}
+1. Press F5 to start debugging which launches your app in Teams App Test Tool using a web browser. Select `Debug in Test Tool (Preview)`.
+1. You can send any message to get a response from the bot.
+
+**Congratulations**! You are running an application that can now interact with users in Teams App Test Tool:
+
+![ai chat bot](https://github.com/OfficeDev/TeamsFx/assets/9698542/9bd22201-8fda-4252-a0b3-79531c963e5e)
+{{/enableTestToolByDefault}}
+{{^enableTestToolByDefault}}
+1. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
```

Comment:
if you haven’t already -> if you haven’t yet?

Reply:
Updated.